installers/source/win/known_issues.rst

```diff
@@ -1,20 +1,22 @@
 * If Boundless Desktop |version| is installed in a custom directory as a sub-folder of
-  the previous install directory, Boundless Desktop older version uninstaller
-  (e.g. 1.1.1) will remove the newest version. However, using the default
-  installation directory poses no issues, and the older versions removal won't
-  affect the latest ones.
-* Boundless Desktop |version| installation changes QGIS project files (qgs
-  and qgz) application mapping, disconnecting them from the previous installation of
-  QGIS. In the same way, uninstalling Boundless Desktop |version| will leave
-  those files unmapped.
-* Running QGIS from a pinned icon in the taskbar won't run scripts in the init_script folder.
+  a previous version's install directory, the uninstaller for the older version of
+  Boundless Desktop (e.g. 1.1.1) will remove the newest version. However,
+  using the default installation directory poses no issue, and removal of older
+  versions won't affect the latest version.
+* Boundless Desktop |version| installation changes application mapping for QGIS
+  project files (qgs and qgz), dissociating them from the previous installation
+  of QGIS. Uninstalling Boundless Desktop |version| after this change in application
+  mapping will leave those files unmapped.
+* Running QGIS from a pinned icon in the taskbar will prevent the execution of scripts
+  in the init_script folder.
 * Uninstaller leaves an empty ``Program Files\Boundless\Desktop\2.0`` directory
-* Uninstaller on Windows 7 will need a system reboot to allow other programs uninstall.
+* Uninstaller on Windows 7 requires a system reboot before other programs can be
+  uninstalled.
 * Uninstallation on user workspace requires administrator permissions.
-* Currently available versions of IPython console plugin (v1.9) is not compatible with
+* Currently available version of IPython console plugin (v1.9) is not compatible with
   IPython 7.2, which is the version deployed with Boundless Desktop. The plugin
```

Comment:
"... is not compatible with the IPython version deployed with Boundless Desktop."

Internal note: the IPython version is actually 7.4, but we should just skip referencing its version here, just note its incompatibility.

Reply:
whoops, i like your _specific_ suggestion better than what i just committed. redoing it